internal/appmetrics/counters.go

```diff
@@ -70,4 +70,12 @@ var (
 		},
 		[]string{"method", "path", "status"},
 	)
+
+	FingerprintRequestCount = promauto.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "devices_api_fingerprint_request_count",
+			Help: "The total number of requests by the Fingerprint",
+		},
+		[]string{"protocol"},
```

Comment:
consider having labels for success and failed states? ie. when return err in consumer.go could increment the request count but with a failure label